components/media-upload-button/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+import { __ } from 'i18n';
+import Button from 'components/button';
+
+class MediaUploadButton extends Component {
```

Comment:
Blocks will be potentially usable outside the editor as well, so I'd still keep it with Editable :)